### ISSUE TITLE
core: Stark logo causes error in Edge and IE due to embedded font-face

### ISSUE THREAD

**Issue by christophercr** [NationalBankBelgium/stark#1115]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#got-a-question-or-problem
</code></pre>

## Current behavior

In Internet Explorer the following error is shown in the console: 

```
CSS3114: @font-face failed OpenType embedding permission check. Permission must be Installable.
Q4VhAPMA0GCSqGSIb3DQEBAQUABIGAHcF3ia6bbyLja+VF2k6RQPCf7zsfJ0pWrDr9qJRqfPecwX97k2BOG8QrV5WUyxbhmmcz0SspE8jsvrxZsQOkKZnsHYiZJId3D5vKFPvU1ElMdA7IPS5vIMkDzejlD9AhObNWGdX7rL2sqTi9sNUMo9ljrbCVtGhYw+LXKf+RpMcAAA==
```

In Edge, the same error is shown but it is even more verbose :s

This is in fact the embedded font face in the Stark logo (`stark_logo_transparent.svg`).

## Expected behavior

No errors related to font-face in the Stark logo should be logged in IE nor Edge.
Ideally we should vectorize the font instead of defining a new base64 font-face.

## Minimal reproduction of the problem with instructions

- run the showcase locally: `npm run showcase`
- open the app in Edge or Internet Explorer
- wait for the app to initialize and then open the console. The error will be logged in there.

## What is the motivation / use case for changing the behavior?

Cross-browser compatibility.

## Environment

<pre><code>
Stark version: 10.0.0-beta.2

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [X] IE version 11
- [X] Edge version 16
</code></pre>

**Comment by cnomes** [comment]
@christophercr @SuperITMan
![dark](https://user-images.githubusercontent.com/11627319/52266991-23615700-2938-11e9-8dfb-f1e37a3d8c73.png)
![light](https://user-images.githubusercontent.com/11627319/52266992-23615700-2938-11e9-8daa-e81be40ce9cd.png)

**Comment by cnomes** [comment]
![light](https://user-images.githubusercontent.com/11627319/52268202-345f9780-293b-11e9-8955-4cf86f8da40d.png)
![dark](https://user-images.githubusercontent.com/11627319/52268200-345f9780-293b-11e9-9c67-b03e752c101c.png)

**Comment by christophercr** [comment]
Closed in #1126
